### PR TITLE
Dt/task/swmaas 1443 refactor lbs sample code

### PR DESCRIPTION
**Pull request by dtran-phunware**
###### Context
The sample app's project organization and structure was not presented in a way indicative of Phunware's coding standard. This PR provides a refactoring to match good coding practices.

######
* Created packages for groupings of the scenarios and put classes within
* Placed drawables in proper folders
* Added proper naming conventions to layout files
* Clean up resource files
* Disable unnecessary backup. Should be resolved in later point release of location SDK.
* Update gradle dependencies
* Update Java Section

**NOTE**: I put all of the java code in on commit just in case we wanted to easily revert.

**NO UI CHANGES*

### CODE REVIEW

**Comment by npike** [comment]
Reviewing

**Comment by npike** [comment]
Don't forget to update the links in https://github.com/phunware/maas-mapping-android-sdk/blob/master/Samples/README.md, as it points to specific Kotlin files for certain sample Scenarios.

**Comment by npike** [comment]
Can you put all of the vector drawables in `drawable-anydpi` and leave just XML drawables (shapes) in `drawable`?  (See https://commonsware.com/blog/2015/12/21/nodpi-anydpi-wtf.html for reference)

**Comment by dtran-phunware** [comment]
@npike updated branch with updates to readme files as discussed.

File: build.gradle [hunk 1]
```diff
@@ -34,7 +34,7 @@ configurations.all {
 }
 
 ext {
-    pwLocationSdkVersion = '3.7.1'
+    pwLocationSdkVersion = '3.7.4'
     pwMappingSdkVersion = '3.9.2'
```
[the remark this answers is not in the document]
Comment (dtran-phunware) [reply build.gradle]: Updated sdk version to 3.9.7

File: Samples/kotlin/build.gradle [hunk 2]
```diff
@@ -21,6 +21,11 @@ android {
         }
     }
 
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
     packagingOptions {
         exclude 'META-INF/lib_release.kotlin_module'
```
Comment (avakkachen) [inline Samples/kotlin/build.gradle]: Can we remove this exclude pls? We don't need it anymore .

Comment (dtran-phunware) [reply Samples/kotlin/build.gradle]: Can do!

**Pull request merged**